presentation/src/main/java/com/sopt/presentation/calendar/navigation/CalendarNavigation.kt

```diff
@@ -16,16 +19,56 @@ fun NavController.navigateCalendar(navOptions: NavOptions? = null) {
     )
 }
 
+fun NavController.navigateInfoScreen(navOptions: NavOptions? = null) {
+    navigate(
+        route = Info,
+        navOptions = navOptions
+    )
+}
+
+fun NavController.navigateDetailScreen(navOptions: NavOptions? = null) {
+    navigate(
+        route = Detail,
+        navOptions = navOptions
+    )
+}
+
 fun NavGraphBuilder.calendarNavGraph(
     paddingValues: PaddingValues,
     navHostController: NavController
 ) {
     composable<Calendar> {
         CalendarRoute(
-            paddingValues = paddingValues
+            paddingValues = paddingValues,
+            navigateToInfoScreen = {
+                navHostController.navigateInfoScreen()
+            }
         )
     }
+
+    composable<Info> {
+        InfoScreen(
+            viewModel = hiltViewModel(),
+            onNextClick = {
+                navHostController.navigateDetailScreen()
+            }
+        )
+    }
+
+    composable<Detail> {
+        DetailScreen(
+            onBackClick = { navHostController.navigateUp() },
+            onNextClick = {
+
+            })
+    }
 }
 
 @Serializable
 data object Calendar : MainTabRoute
+
+@Serializable
+data object Info : MainTabRoute
+
+@Serializable
+data object Detail : MainTabRoute
```

Comment:
P1: 이 화면들은 하단바가 없는 페이지니까 그냥 Route로 써야 합니답!! 그리고 화면 넘어갈 때 이전 화면에서 선택했던 것들을 넘겨 받아야 해서 data class로 수정해주세욥
```suggestion
@Serializable
data class Info(
  val name: String,
  val category: String,
  val time: Int
) : Route

@Serializable
data class Detail(
  val days: List<Int>
) : Route
```

Reply:
P1 : 각 화면 간 전환될 때 넘겨받아야 할 파라미터나 스택에 대해서 생각해보며 개발하면 좋을 것 같아요